src/main/java/club/issizler/okyanus/api/perms/Permissible.java

```diff
@@ -35,16 +35,4 @@ default void removePermission(String perm) {
         Okyanus.getPermissionService().removePermission(this, perm);
     }
 
-    /**
-     * Gets this permissible's identifier
-     * <p>
-     * It can be:
-     * * CONSOLE for the server console
-     * * Player UUID for players
-     * * UNKNOWN if something's bugged
-     *
-     * @return This permissible's identifier
-     */
-    String getIdentifier();
```

Comment:
What would be the alternative to this identifier, and wouldn't removing it right now break the API?